postprocess/crossplane.jsonnet

```
@@ -0,0 +1,26 @@
+/**
+ * Remove generated crossplane manifests if not required
+ */
+local com = import 'lib/commodore.libjsonnet';
+
+local inv = com.inventory();
+local params = inv.parameters.appcat;
+local cms = params.clusterManagementSystem;
```

Can't you do this here?

```suggestion
local cms = com.inventory().parameters.appcat.clusterManagementSystem;
```

---

I will move this change in the next PR when I resolve the importing issues.